### PR TITLE
Install hwloc dev package explicitly after EFA install

### PR DESCRIPTION
**Pull request by rexcsn**
Signed-off-by: Rex <shuningc@amazon.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: recipes/efa_install.rb [hunk 1]
```diff
@@ -57,3 +57,15 @@
...
   not_if { ::Dir.exist?('/opt/amazon/efa') && !efa_gdr_enabled?}
 end
+
+# EFA installer v1.11.0 removes libibverbs-core, which contains hwloc-devel during install
+# hwloc-devel is needed to compile SGE
+# Reinstall hwloc-devel by itself explicitly
+package "install hwloc-devel explicitly" do
+  package_name value_for_platform(
+    'default' => "hwloc-devel",
+    'ubuntu' => { 'default' => "libhwloc-dev" }
```
Comment (yuleiwan) [inline recipes/efa_install.rb:67]: should we also install hwloc-devel/libhwloc-dev on ubuntu and centos platform although there is no failure?

Comment (rexcsn) [reply recipes/efa_install.rb:67]: Problem currently only occurs on AL2, but EFA mentioned that other OSs might start to change to this naming convention that is causing the trouble on AL2.
Proposing to do this for all OSs, just to be safe.

Comment (yuleiwan) [reply recipes/efa_install.rb:67]: Nice!

**Review by yuleiwan — APPROVED** [review]

**Comment by rexcsn** [comment]
Code check passes: https://travis-ci.com/github/rexcsn/aws-parallelcluster-cookbook/builds/208944174
PR travis check stuck, merging manually

**Pull request merged**